modules/python/python-dest.c

```diff
@@ -483,3 +550,19 @@ python_dd_new(GlobalConfig *cfg)
 
   return (LogDriver *)self;
 }
+
+void
+py_destination_init(void)
+{
+#define register_worker_result(value) \
+    PyModule_AddObject(PyImport_AddModule("syslogng"), \
+  #value, int_as_pyobject(value))
+
+  register_worker_result(WORKER_INSERT_RESULT_DROP);
+  register_worker_result(WORKER_INSERT_RESULT_ERROR);
+  register_worker_result(WORKER_INSERT_RESULT_EXPLICIT_ACK_MGMT);
+  register_worker_result(WORKER_INSERT_RESULT_SUCCESS);
+  register_worker_result(WORKER_INSERT_RESULT_QUEUED);
+  register_worker_result(WORKER_INSERT_RESULT_NOT_CONNECTED);
+  register_worker_result(WORKER_INSERT_RESULT_MAX);
```

Comment:
I would not inject destination-related constants to the main syslogng namespace. Could you add them to the destionation object (since we don't have a superclass or a separate module for destinations)?

Reply:
Yes sure! I will do so.

Reply:
I fixed the issue. I added an extra refactor patch to store the references to clear in an array instead explicitely calling them in `_py_free_bindings`. I felt storing the individual objects would have been too much boilerplate.